Client/Frontend/TabContentsScripts/FormAutofillHelper/AddressAutofillPayload.swift

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+// TO DO: FXIOS-7872
```

Comment:
```suggestion
// TODO: FXIOS-7872
```

Reply:
nit: Would be nice to have description alongside the todo FXIOS ticket number

Reply:
Done